.ci/Jenkinsfile_coverage

```diff
@@ -29,4 +32,10 @@ def handleIngestion(timestamp) {
   kibanaCoverage.uploadCoverageStaticSite(timestamp)
 }
 
+def handleFail() {
+  def buildStatus = buildUtils.getBuildStatus()
+  if(params.NOTIFY_ON_FAILURE && buildStatus != 'SUCCESS' && buildStatus != 'ABORTED') {
+    kibanaCoverage.onFailure()
```

Comment:
I think you can just delete the `kibanaCoverage.onFailure` method and just use this here:

```groovy
slackNotifications.sendFailedBuild(
  channel: '#kibana-qa',
  username: 'Kibana QA'
)
```

Reply:
Perfect!  I knew you would be the best reviewer for this!